cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/AemBundleInfoWidget.kt

```diff
@@ -98,7 +100,7 @@ class AemBundleInfoWidget(vertx: Vertx, config: JsonObject) : AsyncWidget(vertx,
 
     override fun updateState() {
         if (url.isNotBlank()) {
-            httpGet("$url/system/console/bundles.json")
+            httpGet("${url.trimEnd('/')}/system/console/bundles.json")
```

Comment:
I thing better solution for this would be to replace any double slash `//` to single slash `/` but inside httpGet() method - something like `prepareURL` method. What do you think?

Reply:
This turns out to be a bit trickier, so I'll roll it back. Parsing URLs is problematic, ideally we'd have to have some more helper functions to handle cases like this.